### PR TITLE
Filter comments by active file

### PR DESCRIPTION
**Pull request by michelcaradec**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

When doing big code reviews (with lots of comments), it can become complicated to locate the comments from the (active) file we are working on.

Issue Number: #92 

## What is the new behavior?

A new "Filter by filename" command (similar to the "Filter comments by commit" #88 feature) will allow to only show the comments of the active file. Switching from one file to another will refresh the comment list.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Icons used for this feature (and also #88) come from <https://github.com/microsoft/vscode-icons>.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This extension allows you to create a code review file you can hand over to a cu
     - [`code-review.priorities`](#code-reviewpriorities)
     - [`code-review.gitDirectory`](#code-reviewgitdirectory)
     - [`code-review.filterCommentsByCommit`](#code-reviewfiltercommentsbycommit)
+    - [`filterCommentsByFilename`](#filtercommentsbyfilename)
```
Comment (d-koppenhagen) [inline README.md:40]:
```suggestion
    - [`code-review.filterCommentsByFilename`](#filtercommentsbyfilename)
```

File: README.md [hunk 2]
```diff
@@ -344,6 +345,16 @@ Define whether to view only the comments from the current commit or not.
 }
 ```
 
+### `filterCommentsByFilename`
```
Comment (d-koppenhagen) [inline README.md:348]:
```suggestion
### `code-review.filterCommentsByFilename`
```

File: package.json [hunk 3]
```diff
@@ -74,20 +74,36 @@
...
         "title": "Code Review: View comments from the current commit",
         "icon": {
           "light": "images/icons/filter-off-light.svg",
           "dark": "images/icons/filter-off-dark.svg"
         }
       },
       {
-        "command": "codeReview.disableFilterByCommit",
+        "command": "codeReview.filterByCommitDisable",
         "title": "CodeReview: View comments from all commits",
```
Comment (d-koppenhagen) [inline package.json:86]:
```suggestion
        "title": "Code Review: View comments from all commits",
```

File: package.json [hunk 4]
```diff
@@ -74,20 +74,36 @@
...
+        "command": "codeReview.filterByFilenameEnable",
+        "title": "Code Review: View comments from the current file",
+        "icon": {
+          "light": "images/icons/filter-filename-off-light.svg",
+          "dark": "images/icons/filter-filename-off-dark.svg"
+        }
+      },
+      {
+        "command": "codeReview.filterByFilenameDisable",
+        "title": "CodeReview: View comments from all files",
```
Comment (d-koppenhagen) [inline package.json:102]:
```suggestion
        "title": "Code Review: View comments from all files",
```

**Comment by d-koppenhagen** [comment]
Hey, great PR. I have seen just a few small things for the docs (see suggestions)l

**Review by d-koppenhagen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (michelcaradec) [reply README.md]: Sorry for the typos, and thank you for the review.

**Pull request merged**